lib/houston/client.rb

```diff
@@ -80,5 +80,16 @@ def unregistered_devices
     def devices
       unregistered_devices.collect{|device| device[:token]}
     end
+
+    def certificate_data
+      return unless ENV['APN_CERTIFICATE']
+      if File.exist?(ENV['APN_CERTIFICATE'])
+        File.read(ENV['APN_CERTIFICATE'])
+      else
+        # Fallback and return the certificate itself; if it isn't readable, the
+        #   env var is likely the whole string version of the cert's `.pem`.
+        ENV['APN_CERTIFICATE']
```

Comment:
@kaneda Another approach I considered was just using a different `ENV` variable for reading in the entire file. Like `ENV["APN_CERTIFICATE_DATA"]` or something. That might make the intent more clear.